src/main/java/model/Bet.java

```diff
@@ -0,0 +1,38 @@
+package model;
+
+import java.io.Serializable;
+
+/**
+ * Feel free to remove this class and create your own.
```

Comment:
do we need this comment

Reply:
Done. But I thought, that we shouldn't delete comment from condition.